tests/unit/introspection-2.tcl

```diff
@@ -2,6 +2,12 @@ proc cmdstat {cmd} {
     return [cmdrstat $cmd r]
 }
 
+proc getlru {key} {
+    set objinfo [r debug object $key]
+    set lruinfo [lindex [split $objinfo " "] 5]
+    return [lindex [split $lruinfo ":"] 1]
```

Comment:
i'd rather parse the string properly, split it by space and colon, and feed into a dictionary. or if we don't do that, let's at least verify the key name is "lru:", so we don't end up getting the wrong field some day and it'll keep the same value on the two lookups and we won't notice.

Reply:
I see you split the string properly now, but I think the function is missing some assertion at the end to make sure the lru was found.

Reply:
Without assertion, if LRU not found, the test still would fail. Anyway, just added a `fail` at the end to provide more clear information.